### PR TITLE
log bootnodes and static nodes list at debug level

### PR DESCRIPTION
**Pull request by macfarla**
at startup, log the counts (at info) and list (at debug level)

### CODE REVIEW

**Comment by macfarla** [comment]
2023-12-11 10:57:24.902+10:00 | main | INFO  | Besu | Configured 1 bootnodes. 
2023-12-11 10:57:26.809+10:00 | main | DEBUG | Besu | Bootnodes = [enode://fcbe9f83218487b3c0b50878193880e6c25cfd86708c0a0bf0ca91f0ce633746a892fe240afa5b9a880b8bca48e8a22704ef937fdda2d7cc63e4d41ed1b417ae@127.0.0.1:30303] 
2023-12-11 10:57:26.847+10:00 | main | WARN  | Besu | Permissions are disabled. Cannot enable PERM APIs when not using Permissions. 
2023-12-11 10:57:26.849+10:00 | main | DEBUG | Besu | Static Nodes file: /Users/sallymacfarlane/workspace/besu-local-nodes/workdir/besu1/data/static-nodes.json 
2023-12-11 10:57:26.851+10:00 | main | DEBUG | StaticNodesParser | StaticNodes file /Users/sallymacfarlane/workspace/besu-local-nodes/workdir/besu1/data/static-nodes.json does not exist, no static connections will be created. 
2023-12-11 10:57:26.851+10:00 | main | INFO  | Besu | Connecting to 0 static nodes. 
2023-12-11 10:57:26.852+10:00 | main | DEBUG | Besu | Static Nodes = []

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java [hunk 1]
```diff
@@ -2120,8 +2120,6 @@ private void configure() throws Exception {
     instantiateSignatureAlgorithmFactory();
 
     logger.info(generateConfigurationOverview());
-    logger.info("Connecting to {} static nodes.", staticNodes.size());
-    logger.trace("Static Nodes = {}", staticNodes);
```
Comment (macfarla) [inline besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:2124]: Note moving the log from here to where the configuration is happening highlights that the updateNetworkConfig() method is called twice. Not sure if that is intentional

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java [hunk 2]
```diff
@@ -3119,9 +3117,14 @@ private EthNetworkConfig updateNetworkConfig(final NetworkName network) {
     if (listBootNodes != null) {
       if (!p2PDiscoveryOptionGroup.peerDiscoveryEnabled) {
         logger.warn("Discovery disabled: bootnodes will be ignored.");
+      } else {
+        logger.info("Configured {} bootnodes.", p2PDiscoveryOptionGroup.bootNodes.size());
```
Comment (Gabriel-Trintinalia) [inline besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java]:
```suggestion
        logger.info("Configured {} bootnodes.", listBootNodes.size());
```

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java [hunk 3]
```diff
@@ -3119,9 +3117,14 @@ private EthNetworkConfig updateNetworkConfig(final NetworkName network) {
...
+        logger.debug("Bootnodes = {}", p2PDiscoveryOptionGroup.bootNodes);
```
Comment (Gabriel-Trintinalia) [inline besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java]:
```suggestion
        logger.debug("Bootnodes = {}", listBootNodes);
```

**Review by Gabriel-Trintinalia — APPROVED** [review]
lgtm

**Pull request merged**